### DISCUSSION TITLE
Problem with exporting Header component

### DISCUSSION THREAD

**Discussion by DennisWarchocki** [neuefische/questions#264] [Web] [answered]
Hello everyone,

I tried to nest all my elements that im using regulary as a header inside a component.
<img width="571" alt="Bildschirmfoto 2022-10-13 um 21 00 48" src="https://user-images.githubusercontent.com/109968361/195685912-5e0e3a0d-1f62-4195-884b-938f9133880a.png">
   
So I tried to return here the new Header component that I've created before, and I get no Problems displayed, everything looks fine. 
<img width="866" alt="Bildschirmfoto 2022-10-13 um 21 06 07" src="https://user-images.githubusercontent.com/109968361/195686533-2d754a42-9128-4deb-9a50-49b5427bae57.png">

But when I try to route to that page I get that Error Message, and I cant understand why, any suggestions?

<img width="947" alt="Bildschirmfoto 2022-10-13 um 21 09 07" src="https://user-images.githubusercontent.com/109968361/195687707-b97c7497-f3dd-4aef-8ed1-6e49777513c8.png">

repo:
https://github.com/DennisWarchocki/capstone-project/pull/14

**Comment by mbosselmann** [comment, marked as answer]
Hi Dennis,
I think your import of your `StyledNavbar` is wrong as this components lies in your `StyledNew.js` and not in your `StyledNav.js`. :)

Does this help?

**Reply by DennisWarchocki** [reply]
Thank you. my Code didnt show up any Problems and I was really confused. It was just the import because i moved the styling out of this file. 
👍
